utilities/helpers/updateSpecificFieldsHelper.js

```diff
@@ -135,6 +130,95 @@ const update = async (author, permlink, authorPermlink, voter) => {
   }
 };
 
+const updateWobjParent = async ({ authorPermlink, parent }) => {
+  await Wobj.update({ author_permlink: authorPermlink }, { parent });
+  await updateMapFromParent(authorPermlink, parent);
+};
+
+const processingParent = async (authorPermlink) => {
+  const { app: { admins } } = await App.getOne({ name: config.app });
+  const { wobjects: [{ fields } = {}] } = await Wobj.getSomeFields('parent', authorPermlink);
+  // check for admin votes
+  for (const field of fields) {
+    for (const vote of field.active_votes) {
+      if (_.includes(admins, vote.voter)) {
+        return handleAdminVoteOnParent({ authorPermlink, fields, admins });
+      }
+    }
+  }
```

Comment:
what if there is more than one admin like on 1 field?

Reply:
+